quipucords/scanner/network/connect.py

```diff
@@ -235,7 +235,7 @@ def _connect(manager_interrupt,
              credential,
              connection_port,
              use_paramiko=False,
-             forks=50,
+             forks=25,
```

Comment:
In a way, we are defaulting this to 25 twice - which isn't necessarily terrible - but it does mean that if we ever update it in the future, we have to unnecessarily change it in 2 places instead of one. The function that calls this function defaults forks to whatever is returned by get_default_forks() so we could remove this default here if we wanted - not sure if it is worth the  trouble. Thoughts?